azimuth/modules/base_classes/module.py

```diff
@@ -200,18 +201,18 @@ def _get_table_key(self) -> Optional[PredictionTableKey]:
 
     def get_threshold(self) -> Optional[float]:
         # The default is None so we have to handle it this way.
-        current_pipeline = self.get_current_pipeline()
+        current_pipeline = self.get_current_pipeline_definition()
 
         thresh = self.mod_options.threshold
         if thresh is None:
             thresh = current_pipeline.threshold
         return thresh
 
-    def get_current_pipeline(self) -> PipelineDefinition:
-        """Get current pipeline for this spec.
+    def get_current_pipeline_definition(self) -> PipelineDefinition:
```

Comment:
Getters usually get the "current" thing, unsell specified otherwise. Is there a reason for specifying "current" here?
```suggestion
    def get_pipeline_definition(self) -> PipelineDefinition:
```
Also, the type really is `PipelineDefinition`, so I wouldn't hesitate to call the variables `pipeline_definition` too, for example:
```diff
-        current_pipeline = self.get_current_pipeline_definition()
+        pipeline_definition = self.get_pipeline_definition()
```

Reply:
I agree for `current`, good call. As for the `get`, I would be tempted to stay consistent with the other methods, such as `get_threshold`.  And also given that it is a method and not a property. Thoughts?

Reply:
I agree. I did not suggest removing `get_`.

Reply:
Oh sorry, I read the comment too quickly it seems!

Reply:
I figured. No worries!